packages/zosfiles/__tests__/__system__/cli/mount/fs/cli.files.mount.fs.system.test.ts

```diff
@@ -0,0 +1,169 @@
+        });
+
+        afterEach(async () => {
+            await TestEnvironment.cleanUp(TEST_ENVIRONMENT_NO_PROF);
+        });
+
+        it("should mount a file system", () => {
+            // if API Mediation layer is being used (basePath has a value) then
+            // set an ENVIRONMENT variable to be used by zowe.
+            if (defaultSys.zosmf.basePath != null) {
+                TEST_ENVIRONMENT_NO_PROF.env[ZOWE_OPT_BASE_PATH] = defaultSys.zosmf.basePath;
+            }
+
+            let response = runCliScript(__dirname + "/__scripts__/command/command_mount_fs_fully_qualified.sh",
+                TEST_ENVIRONMENT_NO_PROF,
+                [fsname, mountPoint,
+                    defaultSys.zosmf.host,
+                    defaultSys.zosmf.port,
+                    defaultSys.zosmf.user,
+                    defaultSys.zosmf.pass]);
+            expect(response.stderr.toString()).toBe("");
+            expect(response.status).toBe(0);
+
+            response = runCliScript(__dirname + "/__scripts__/command/command_list_fs_fully_qualified.sh",
+                TEST_ENVIRONMENT_NO_PROF,
+                [fsname,
+                    defaultSys.zosmf.host,
+                    defaultSys.zosmf.port,
+                    defaultSys.zosmf.user,
+                    defaultSys.zosmf.pass]);
+            expect(response.stderr.toString()).toBe("");
+            expect(response.status).toBe(0);
+            expect(response.stdout.toString()).toContain(mountPoint.slice(1));
+        });
+
+        it("should unmount a file system", () => {
+            // if API Mediation layer is being used (basePath has a value) then
+            // set an ENVIRONMENT variable to be used by zowe.
+            if (defaultSys.zosmf.basePath != null) {
+                TEST_ENVIRONMENT_NO_PROF.env[ZOWE_OPT_BASE_PATH] = defaultSys.zosmf.basePath;
+            }
+
+            let response = runCliScript(__dirname + "/__scripts__/command/command_unmount_fs_fully_qualified.sh",
+                TEST_ENVIRONMENT_NO_PROF,
+                [fsname,
+                    defaultSys.zosmf.host,
+                    defaultSys.zosmf.port,
+                    defaultSys.zosmf.user,
+                    defaultSys.zosmf.pass]);
+            expect(response.stderr.toString()).toBe("");
+            expect(response.status).toBe(0);
+
+            response = runCliScript(__dirname + "/__scripts__/command/command_list_fs_fully_qualified.sh",
+                TEST_ENVIRONMENT_NO_PROF,
+                [fsname,
+                    defaultSys.zosmf.host,
+                    defaultSys.zosmf.port,
+                    defaultSys.zosmf.user,
+                    defaultSys.zosmf.pass]);
+            expect(response.stderr.toString()).toBeTruthy();
+            expect(response.status).toBe(1);
+        });
+    });
+
+    describe("With profiles", () => {
+        it("should mount a file system", () => {
```

Comment:
Same comment here as the other test. They should be self contained and not depend on one another.